tests/eigenwave3d.py

```diff
@@ -156,14 +157,14 @@ def default(compiler=None, execute=False, nthreads=1,
         f.write(str(tile))
         f.close()
     domain_size = (1.0, 1.0, 1.0)
-    grid_size = (100, 100, 100)
+    grid_size = (247, 247, 247)
```

Comment:
Note that travis is currently timing out - likely because of larger domain size.